app/agents/RecruitmentManagerAgent.py

```diff
@@ -243,10 +242,32 @@ async def check_current_priority(self):
 
         if should_end:
             self.agent.logger.info(
-                "All recruitment stages are DONE. Ending StageCommunication behaviour..."
+                "All recruitment stages are DONE. Ending AgentCommunication behaviour..."
             )
             self.kill()
 
+    async def handle_analysis_result(self, data: List[str]):
+        self.agent.logger.info(
+            f"Received message with CV analysis result equal to {data[0]}"
+        )
+
+        recruitments = self.agent.recruitment_module.get_by_job_and_candidate(
+            self.agent.job_offer_id, 
+            self.agent.candidate_id,
+        )
+        if len(recruitments) == 0:
+            f"No recruitments with job_offer_id: {self.agent.job_offer_id} and candidate_id: {self.agent.candidate_id} found."
+            return
+        
+        result = self.agent.recruitment_module.update(
+            recruitments[0]._id, 
+            {"application_rating": int(data[0])}
+        )
+        if result:
+            self.agent.logger.info(f"Successfully saved the candidate {self.agent.candidate_id} rating for job offer {self.agent.candidate_id}")
+        else:
+            self.agent.logger.info(f"Failed to save the candidate {self.agent.candidate_id} rating for job offer {self.agent.candidate_id}")
+
```

Comment:
Tutaj chyba powinno być `for job offer {self.agent.job_offer_id}`

Reply:
poprawiłem na swoim branchu. Zamykam